vnext/Microsoft.ReactNative.Cxx/JSValueWriter.h

```diff
@@ -23,6 +23,8 @@ template <class T, std::enable_if_t<std::is_convertible_v<T, std::string_view>,
 void WriteValue(IJSValueWriter const &writer, T const &value) noexcept;
 template <class T, std::enable_if_t<std::is_convertible_v<T, std::wstring_view>, int> = 1>
 void WriteValue(IJSValueWriter const &writer, T const &value) noexcept;
+void WriteValue(IJSValueWriter const &writer, winrt::hstring value) noexcept;
+void WriteValue(IJSValueWriter const &writer, const winrt::param::hstring value) noexcept;
```

Comment:
>const winrt::param::hstring [](start = 46, length = 27)

Parameter must not be const on its own. It unnecessary restricts access to a variable passed by value.
Here and and in the line above we must use the 'const &' as we do in other places.
Though, first I want to understand if we need these two methods.

Reply:
Please scratch the last question about if we need it or not: we need it.
The original question is still: please either remove the `const` or use `const &`